src/javascripts/Services/gridService.js

```diff
@@ -0,0 +1,90 @@
+// Base view to show nodes and columns. It's like a tree view but with no depth. 
+// Utilized by all views. Functions for specialized views (e.g. treeView/recordView)
+// will overwrite most of these base functions.
+ngapp.service('gridService', function($timeout, htmlHelpers) {
+    this.buildFunctions = function(scope, element) {
+        // scope functions
+        scope.reload = function() {
+            if (!scope.tree) return;
+            let oldTree = scope.tree;
+            scope.clearSelection(true);
+            scope.buildTree();
```

Comment:
should the function really be `buildTree` if we have a grid in this case?  just a thought, might not be worth it to change at the moment.